src/components/templates/Poster/components/IntroVideoPreviewModal/IntroVideoPreviewModal.scss

```diff
@@ -0,0 +1,30 @@
+@import "scss/constants";
+
+$iframe-width: 50vw;
+$iframe-height: calc(#{$iframe-width} * 9 / 16);
+$optimal-margin: 20px;
+
+.IntroVideoPreviewModal {
+  &__title {
+    font-size: 16px;
+    margin-bottom: $optimal-margin;
+    font-weight: 500;
+  }
+
+  &__iframe {
+    width: $iframe-width;
+    height: $iframe-height;
+    border: none;
+    border-radius: 15px;
+    background-color: $lighter-intermediate-grey;
+  }
+
+  .modal-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    background: none;
+    box-shadow: none;
+  }
```

Comment:
I noticed that the `Modal` component had the `centered` prop set on it, do we need all of these style overrides here? Can they be controlled through the `Modal` components props instead (as it's 'first class public API')?

Reply:
`centered` prop refers to a modal in the window space, not the content of the modal.
![image](https://user-images.githubusercontent.com/63194656/116594967-7796ee00-a92b-11eb-8516-2ad6bd375c0e.png)


The styles here override the default Modal styles (according to the design).

Even though the declared public API has `contentClassName`
![image](https://user-images.githubusercontent.com/63194656/116594777-43bbc880-a92b-11eb-894f-6dd54f1c2c4f.png)

If you add it to the <Modal/> component
![image](https://user-images.githubusercontent.com/63194656/116594823-50402100-a92b-11eb-8d6a-af940a665381.png)

It doesn't assign this to `modal-content div`, but rather sets it as an HTML attribute
![image](https://user-images.githubusercontent.com/63194656/116594919-6a79ff00-a92b-11eb-85ca-aef9bc34aaec.png)

Reply:
Linking to the docs is generally better than screenshots. I'm assuming your screenshots were from the following?

- https://react-bootstrap.github.io/components/modal/
  - https://react-bootstrap.github.io/components/modal/#modal-props
  
If so, that seems to be the docs for `v1.5.3`, whereas we appear to be using `1.0.1`:

- https://github.com/sparkletown/sparkle/blob/staging/package.json#L50
  -  "react-bootstrap": "^1.0.1",

This section of the docs page is defined at:

- https://github.com/react-bootstrap/react-bootstrap/blob/v1.0.1/www/src/pages/components/modal.js#L147

But it references `ComponentApi` with `data.Modal` and isn't easy to find the docs.

So looking at the component itself, we can see the props that it has:

- https://github.com/react-bootstrap/react-bootstrap/blob/v1.0.1/src/Modal.js

It doesn't have the `contentClassName` prop, which is why it was just passed straight through to the HTML.

---

Based on the above, it would be good to add a `@debt` comment saying to replace it's usage with the `contentClassName` prop when we upgrade to a version of react-bootstrap that supports it.

Reply:
Added in https://github.com/sparkletown/sparkle/pull/1299/commits/fef34d310713210aef00bbc6ad1367f651ef0fd8